infra/dynamodb.tf

```diff
@@ -0,0 +1,20 @@
+resource "aws_dynamodb_table" "basic-dynamodb-table" {
```

Comment:
Best to choose a cleaner resource name. Since the table is for user profiles, we could call the resource 'user_profiles' instead.